js/viewer/annotation/annotation-group.js

```diff
@@ -97,13 +97,22 @@ function AnnotationGroup(id, anatomy, description, parent){
 
         var rst = ["<g id='"+this.id+"'>"];
         var i;
+        var hasValidAnnotations = false;
         for(i = 0; i < this.annotations.length; i++){
```

Comment:
I don't see the point of this extra `hasValidAnnotations`, you could change this function starting from line 98 to be

```javascript
var rst = [], content;

this.annotations.forEach(function (annot) {
  content = annot.exportToSVG();
  if (content != "") {
    rst.push(content);
  }
});

if (rst.length === 0) {
  return "";
}

return "<g id='" + this.id + "'>" + rst.join("") + "</g>";
```

Reply:
Changed the function.